godot-codegen/src/context.rs

```diff
@@ -50,6 +51,10 @@ impl<'a> Context<'a> {
         ctx
     }
 
+    pub fn get_engine_class(&self, class_name: &TyName) -> Option<&Class> {
+        self.engine_classes.get(class_name).copied()
+    }
```

Comment:
Is there a case where `get_engine_class` returns `None`?
Otherwise, consider returning a reference directly.

Reply:
:+1: